handbook/engineering/core-application/playbooks/fix-gitserver-eviction-on-macos.md

```diff
@@ -0,0 +1,63 @@
+# Fix gitserver eviction on macOS
+
+Finder and About This Mac will report used disk bytes as available if the space is occupied by Time Machine [local snapshots](https://support.apple.com/en-us/HT204015). This differs from how `df` and Disk Utility behave — both would report that space as occupied.
+
+When disk space is low, gitserver will begin evicting repositories until the minimum threshold of `SRC_REPOS_DESIRED_PERCENT_FREE` has been met. This can lead to a confusing situation for developers where gitserver's view of the disk is that it is low on capacity, but the operating system indicates that sufficient space is available.
+
+> Note: these snapshots exist locally because the Time Machine backup disk was not available. It is recommended that you properly back up your system prior to using this playbook.
+
+## Playbook
+
+If gitserver is evicting repositories due to disk pressure, first use `df` to determine how much space is available. These results will be consistent with what gitserver believes to be true about the disk.
+
+```
+$ df -h
+Filesystem      Size   Used  Avail Capacity iused      ifree %iused  Mounted on
+/dev/disk1s1   932Gi   10Gi  316Gi     4%  488304 9767489856    0%   /
```

Comment:
Perhaps it would help to supply an example of a disk that is almost at capacity, such as the numbers I was seeing?